src/box/lua/tuple.c

```diff
@@ -687,6 +687,42 @@ luaT_pushtuple(struct lua_State *L, box_tuple_t *tuple)
 	luaL_setcdatagc(L, -2);
 }
 
+/**
+ * Push to Lua stack a table with the information about a tuple, located on top
+ * of the stack.
+ */
+static int
+lbox_tuple_info(lua_State *L)
+{
+	int argc = lua_gettop(L);
+	if (argc != 1)
+		luaL_error(L, "Usage: tuple:info()");
+
+	struct tuple_info tuple_info;
+	struct tuple *tuple = luaT_checktuple(L, 1);
+	struct tuple_format *format = tuple_format_by_id(tuple->format_id);
+	format->vtab.tuple_info(format, tuple, &tuple_info);
```

Comment:
Nit: It'd be nice to have a helper function `tuple_info(tuple, info)`.

Reply:
Done.